pygments/formatters/html.py

```diff
@@ -933,16 +933,26 @@ def _highlight_lines(self, tokensource):
             else:
                 yield 1, value
 
-    def wrap(self, source, outfile):
+    def wrap(self, source, *, include_div):
```

Comment:
I see `outfile` in a subclass in the documentation example at

https://pygments.org/docs/formatters/#formatter-classes

Is this something to be concerned about?

Reply:
I don't think it should be there given it's unused. I'll update the docs though to match.

Reply:
It looks like it's only used there, and even in the sample code, `outfile` was unused. That said, looking at the wrapping code, I think I'll start a `Html5Formatter` or similar and redo all of this, because the whole table business shouldn't be necessary at all ... but for now the comments are now consistent with the code, and it should only affect users who have subclassed the `HtmlFormatter` _and_ reimplemented `wrap`.